### DISCUSSION TITLE
count totalLikes per post with supabase-js

### DISCUSSION THREAD

**Discussion by schweppes-0x** [supabase/supabase#9415] [Questions] [answered]
I've been using the supabase-js for my application and I stumbled upon a problem. 
I want to get a table like this 

| Post | TotalLikes |
| -------- | -------------- |
| 0001    | 3           |
| 0002   | 12          |
| 0005   | 0          |

The SQL-query for this would be:
```
SELECT Post, COUNT(*) as TotalLikes from "Votes"
where positive_vote = true
GROUP BY Post
```

I found this stackoverflow: https://stackoverflow.com/questions/65612167/how-to-get-count-in-supabase

however this does not help in this case. 

Any help/suggestion is welcome

**Comment by steve-chavez** [comment, marked as answer]
You can't do GROUP BY directly through `supabase-js` right now. You can use an [rpc](https://supabase.com/docs/reference/javascript/rpc) as a workaround.
